### PR TITLE
Created time-map benchmark script

### PR DESCRIPTION
**Pull request by arnasbr**

### CODE REVIEW

File: scripts/time-map.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    });
+    sleep(1);
+}
+
+function generateBody() {
+    return JSON.stringify({
+        arrival_searches: {
+            one_to_many:[
+                {
+                    id: 'Time map fast benchmark',
```
Comment (Donatas-L) [inline scripts/time-map.js:34]: Time map fast benchmark? 

It's just time map

File: scripts/time-map.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    coords: generateRandomCoordinate(51.50750,  51.50950, -0.128015, -0.128615),
+                    arrival_time_period: 'weekday_morning',
```
Comment (Donatas-L) [inline scripts/time-map.js:36]: This is ifalcon request @danielnaumau

Comment (danielnaumau) [reply scripts/time-map.js:36]: yes, I see

**Review by danielnaumau — CHANGES_REQUESTED** [review]

**Comment by Donatas-L** [comment]
@arnasbr https://docs.traveltime.com/api/reference/isochrones

**Pull request closed without merging**